### PR TITLE
Update workflows to use non-deprecated versions.

### PR DESCRIPTION
**Pull request by SanjayVas**

### CODE REVIEW

File: .github/workflows/build-test.yml [hunk 1]
```diff
@@ -25,13 +25,43 @@ on:
...
     steps:
-      - name: Check out revision
-        uses: actions/checkout@v2
-
-      - name: Bazel build and test
-        uses: world-federation-of-advertisers/actions/bazel-build-test@v1
-        with:
-          workspace-path: .
-          
+    - uses: actions/checkout@v3
```
Comment (YuhongWang-Amazon) [inline .github/workflows/build-test.yml:30]: do we still need the name for this step?

Comment (SanjayVas) [reply .github/workflows/build-test.yml:30]: No. The name is just for the visual representation. It just defaults to the action if you don't have one, and `actions/checkout@v3` carries enough meaning.

**Comment by wfa-reviewable** [comment]
This change is [Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/consent-signaling-client/42)

**Review by YuhongWang-Amazon — APPROVED** [review]
> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/consent-signaling-client/42)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @SanjayVas)

**Pull request merged**